changes/2334.1.feature.rst

```diff
@@ -0,0 +1 @@
+Briefcase Dev command can now accept platforms arguments
```

Comment:
Right now, this should probably be a `misc` rather than feature, as there's no "user facing" manifestation of this command. When we add an implementation for `briefcase dev web`, *that* will be a feature.
```suggestion
Development mode behavior can now be platform-specific.
```